src/rules/resolutions.ts

```diff
@@ -15,9 +15,24 @@ import * as core from '@actions/core';
 import { PackageJson } from 'type-fest';
 
 export const validateResolutions = (packageJson: PackageJson) => {
-  if (packageJson.resolutions) {
+  const ignoredResolutions = core.getMultilineInput('ignore-resolutions');
+
+  if (packageJson.resolutions && !ignoredResolutions.length) {
     core.setFailed(
       'Resolutions may not be set. Please investigate the root cause of your dependency issues!'
     );
   }
+
+  if (packageJson.resolutions && Array.isArray(ignoredResolutions)) {
```

Comment:
Nit pick, but we already know `ignoredResolutions` is an array

Reply:
Yes, this is another leftover. Will amend 👍 
I actually already changes this on my other branch with additional changes.